tests/dialects/test_triggers.py

```diff
@@ -59,3 +59,58 @@ def test_drop_triggers(self):
             in QueryPool.queries[-2]
         )
         assert 'DROP FUNCTION ' in QueryPool.queries[-1]
+
+@pytest.mark.skipif('not uses_native_versioning()')
+class TestTriggerSyncingCustomTableNameFormat(object):
```

Comment:
This set of tests is copied from above - but I set it up using a "specially named" versions table instead of a default one